tank/resnet50_torch/resnet50_torch_test.py

```diff
@@ -88,6 +88,9 @@ def test_module_dynamic_gpu(self):
     @pytest.mark.skipif(
         check_device_drivers("vulkan"), reason=device_driver_info("vulkan")
     )
+    @pytest.mark.xfail(
```

Comment:
Instead of xfailing can we skip it if it's running on the mac-m1. I believe it runs on the other Vulkan backends.

Reply:
@pashu123 I've marked the tests xfail only for vulkan on macOS as suggested.